ssm/ssm.go

```diff
@@ -0,0 +1,117 @@
+package ssm
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/roboll/helmfile/helmexec"
+	"go.uber.org/zap"
+)
+
+var (
+	svcSSM  *ssm.SSM
+	ssmPath string
+	logger  *zap.SugaredLogger
+)
+
+func Run() {
+	logger = helmexec.NewLogger(os.Stdout, "debug")
+	ssmPath, exists := os.LookupEnv("SSM_PATH")
```

Comment:
Ideally we could support multiple paths. Perhaps use `;` delimiter like other `PATH` vars

Reply:
Definitely think we should have the ability to support multiple paths. 

Would you rather define each path in the environment? Or just directly in the helmfile? I think it makes more sense to just put right in the helmfile.

Reply:
I'm looking at updating the helmfile.yml spec to include `ssm:` at the root level of the YAML.

```yaml
ssm:
  region: us-west-2
  prefix: /path/in/ssm
```

The idea is then the prefix is optional when you start using ssm directly

```
{{ ssm "postgres_pass" }}
```

instead of 

```
{{ ssm "/path/in/ssm/postgres_pass" }}
```

But of course you can do something like..

```
{{ ssm "/path/one/postgres_pass" }}
{{ ssm "/path/two/redis_pass" }}
```